src/iosource/Packet.h

```diff
@@ -74,7 +74,7 @@ class Packet {
      */
     Packet(int link_type, pkt_timeval* ts, uint32_t caplen, uint32_t len, const u_char* data, bool copy = false,
            std::string tag = "") {
-        Init(link_type, ts, caplen, len, data, copy, tag);
+        Init(link_type, ts, caplen, len, data, copy, std::move(tag));
```

Comment:
I've come across `tag` in another context. It seems completely unused in-tree. Maybe we can deprecate it separately assuming #3431 will provide a different approach.